internal/pkg/template/input/file.go

```diff
@@ -82,7 +82,7 @@ func (i file) validate() error {
 		{
 			Tag:          "template-input-id",
 			Func:         validateInputId,
-			ErrorMessage: "{0} can only contain alphanumeric characters, dots and underscores",
+			ErrorMessage: "{0} can only contain alphanumeric characters, dots, underscores and dashes",
```

Comment:
Povodne som nechcel v `input.Id` znak `-` (minus), ale nakoniec to nevadi, `Input("my-input")`, tak som to zjednotil s `ConfigId/ConfigRowId`, kde tiez moze byt `-`.